optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/employee/EmployeeRestServiceImpl.java

```diff
@@ -122,4 +123,29 @@ public Boolean removeEmployeeAvailability(Integer tenantId, Long id) {
         return true;
     }
 
+    @Override
+    @Transactional
+    public Boolean updateEmployee(Integer tenantId, Long id, Employee newValue) {
+        Employee employee = entityManager.find(Employee.class, id);
```

Comment:
This should use the EntityManager.merge() functionality, which returns a new instance.
There's a bunch of important theory behind this, I can explain in a conf call.

This is stopping the PR merge for now.